msvc/libmini.targets

```diff
@@ -6,4 +6,5 @@
   <Import Project="$(MSBuildThisFileDirectory)libmini-arch.targets" />
   <Import Project="$(MSBuildThisFileDirectory)libmini-interp.targets" />
   <Import Project="$(MSBuildThisFileDirectory)libmini-llvm.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)libmini-netcore.targets" />
```

Comment:
I forgot to make this conditional...somehow. What is the preferred way?

Reply:
if you didn't know, the convention is to put them in an itemgroup with a Condition= on it

Reply:
That's what I would normally do. However, in this context it is problematic. The variable you'd test in the condition could be set from three different places: a) MSBuild command line b) mono.props file (and VS IDE by proxy) c) by reverse lookup from config.h generated by `./configure` script. The first two are easy to handle, but the last one is late-bound and currently obtained in very complex way.

Reply:
Can you define late-bound for me? I'd like to help you solve this in a clear and understandable way.

Reply:
It's an output of a task parameter. I replicated the whole logic used for LLVM in https://github.com/mono/mono/pull/14418/commits/bab126ce29b01ba72b70dca5c375cade523ad964, but it is cumbersome and in the end I reverted it at @lateralusX's request.

I do know ways how to solve it more elegantly but they are way out of scope for this change where I just wanted to get the basic build working. I didn't understand why a simple `Condition="'$(MONO_ENABLE_NETCORE)' == 'true'"` would not work but after figuring out the whole logic in `mono.external.targets` I came to the conclusion that trying to rework it at this point would result in too big change.